src/prommis/uky/uky_flowsheet.py

```diff
@@ -1629,7 +3053,7 @@ def add_costing(flowsheet):
     )
 
     feed_input = units.convert(
```

Comment:
On line 3365, the code includes:
```
m.fs.feed_input.fix()
m.fs.feed_grade.fix()
m.fs.recovery_rate_per_year.fix()
m.fs.solid_waste.fix()
m.fs.precipitate.fix()
m.fs.dust_and_volatiles.fix()
m.fs.power.fix()
```
These were added to the original costing tests because the variables were created from scratch and should be fixed during the full solve. However, these quantities should be calculated already by the UKy flowsheet, so it would be good to couple them with something like:

```
m.fs.feed_input = Var(initialize=reasonable_initial_guess, units=correct_units)
m.fs.feed_input_constraint = Constraint(expr=(m.fs.feed_input = units.convert(flowsheet_variable_or_expression, to_units=correct_units)))
```

for each of the variables. This will help ensure no DOF are changing, as well.